device/mellanox/x86_64-mlnx_msn2700-r0/ACS-MSN2700/pg_profile_lookup.ini

```diff
@@ -0,0 +1,17 @@
+# PG lossless profiles.
+# speed cable size    xon  xoff threshold
+  10000  5m   34816  18432 16384  1
+  25000  5m   34816  18432 16384  1
+  40000  5m   34816  18432 16384  1
+  50000  5m   34816  18432 16384  1
+ 100000  5m   36864  18432 18432  1
+  10000  40m  36864  18432 18432  1
+  25000  40m  39936  18432 21504  1
+  40000  40m  41984  18432 23552  1
+  50000  40m  41984  18432 23552  1
+ 100000  40m  54272  18432 35840  1
+  10000  300m 49152  18432 30720  1
+  25000  300m 71680  18432 53248  1
+  40000  300m 94208  18432 75776  1
+  50000  300m 94208  18432 75776  1
+ 100000  300m 184320 18432 165888 1
```

Comment:
I didn't see matching code using this look up table. Who is using it where and how?

Reply:
it is passed as an argument for `buffermgrd`: https://github.com/Azure/sonic-buildimage/pull/1250/files#diff-849a3366595afffe34a13f4a543dca56